src/quantum/azext_quantum/commands.py

```diff
@@ -94,6 +98,30 @@ def one(key, value):
         histogram = results['histogram']
         return [one(key, histogram[key]) for key in histogram]
 
+    elif 'errorData' in results:
+        notFound = 'Not found'
+        status = notFound
+        errorCode = notFound
+        errorMessage = notFound
+        target = notFound
+        jobId = notFound
+        submissionTime = notFound
+        if 'status' in results:
```

Comment:
Might be better to use [inline ternary operator](https://pythonexamples.org/python-ternary-operator/) instead of these `if` statements

Reply:
Thanks, XField, I really appreciate you pointing out the Python ternary operator, and I will definitely keep that construct in mind for future Python adventures, but in this instance I’d like to go with Zelin’s suggestion below to use the dict.get() function.